_posts/cas/2017-07-05-cas-contribution-guide.md

```diff
@@ -108,6 +110,14 @@ In order to successfully finish this exercise you need:
 2. [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), eclipse or NetBeans (Depending on the change, `vim` may be perfectly fine too)
 3. [Java (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 
+## File Contributor License Agreements
```

Comment:
I understand [current Apereo policy](https://www.apereo.org/licensing/agreements) to require that all **contributions** be via CLA.

> The Apereo Foundation requires that all project contributors complete a Contributor License Agreement (CLA).

(I have [proposed relaxing Apereo ICLA requirement policy](https://github.com/apetro/less-iclas-more-better) and personally I'd favor a policy of [not requiring ICLAs of anyone](https://sfconservancy.org/blog/2014/jun/09/do-not-need-cla/), but thusfar have not gained traction.)